app/services/feeds/import.rb

```diff
@@ -1,13 +1,24 @@
 module Feeds
+  # Responsible for fetching RSS feeds for multiple users.
+  #
+  # @see {Feeds::Import.call}
   class Import
-    def self.call(users: nil, earlier_than: nil)
-      new(users: users, earlier_than: earlier_than).call
+    # Fetch the feeds for the given users (with some filtering based on internal business logic).
+    #
+    # @param users_scope [ActiveRecord::Relation<User>] the initial scope for determining the users
```

Comment:
ah - this is a relation. We're not taking a named scope symbol and sending it to User. I'm sure this was all clear in your head but "scope" has a few meanings and my mind wandered down the wrong path for a minute. 

The default is `User` which is just the plain old unscoped set.

Makes sense.